broadcast/src/buffered/ingress.rs

```diff
@@ -34,18 +40,40 @@ impl<D: Digest, M: Digestible<D>> Mailbox<D, M> {
 }
 
 impl<D: Digest, M: Digestible<D>> Mailbox<D, M> {
-    /// Get a message by digest.
-    pub async fn get(&mut self, digest: D) -> oneshot::Receiver<M> {
+    /// Wait for a message by digest.
+    pub async fn wait(&mut self, digest: D) -> oneshot::Receiver<M> {
```

Comment:
The name works, but don't love it... maybe one of the following?
- `wait_for()`
- `get_or_await()`